src/main/java/uk/gov/hmcts/dts/mytime/entities/LeaveRequest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.dts.mytime.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.dts.mytime.models.LeaveStatus;
+import uk.gov.hmcts.dts.mytime.models.LeaveType;
+
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "leave_request")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(exclude = {"id"})
+public class LeaveRequest {
+    @Id
+    @SequenceGenerator(name = "seq-gen", sequenceName = "leave_request_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq-gen")
+    @Column(insertable = false, updatable = false, nullable = false)
+    private int id;
+
+    @Column(nullable = false)
+    private Integer employeeId;
+
+    @Column(nullable = false)
+    private Integer approverId;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private LeaveType type;
+
+    @Enumerated(EnumType.STRING)
+    private LeaveStatus status;
+
+    @Column(nullable = false)
+    private LocalDate startDate;
```

Comment:
Consider if we want LocalDateTime for half day (i.e 9 till 1 am or 1 till 5 or w/e pm)

Or if we want another field added here for a half day am or pm tick box? As discussed.

Or if we want to do that later on another ticket :)

Reply:
We add another ticket and do it there :)